concepts/90 Integration Guides/30 Create a DevExtreme application with Vite.js.md

```diff
@@ -0,0 +1,160 @@
+    cd vite-project; npm install
+
+To start the development server, execute the following command:
+
+<!--Shell-->
+    npm run dev
+
+![Vite server up and running](/images/Vite/vite-ready.png)
+
+## 2. Configure DevExtreme dependencies
+
+Add the following dependencies to the `package.json` file:
+
+<!--JSON-->
+    "dependencies": {
+        "devextreme": "^23.2",
+        "devextreme-vue": "^23.2",
+        ...
+    }
+
+Install the new dependencies:
+
+<!--Shell-->
+    npm install
+
+## 3. Remove unnecessary code
+
+1. Open the `srtc/App.vue` file, and replace its content with the following two tags:
+
+    <!--TypeScript-->
+        <script setup lang="ts">
+        </script>
+
+        <template>
+        </template>
+
+2. Remove the `src/components/HelloWorld.vue` file because we no longer need it.
+
+3. The `src/style.css` file includes CSS code that overrides DevExtreme styles. Open the file and remove its contents.
+
+## 4. Set up a DevExtreme theme
+
+1. Add the following `import` statement to the `src/main.ts` file:
+
+    <!--TypeScript-->
+        import 'devextreme/dist/css/dx.fluent.blue.light.css';
+
+    This statement applies the `fluent` theme to your application. You can select a different DevExtreme theme if you wish.
+
+2. To apply this theme throughout your application, edit the `index.html` file in the project's root directory. Add the `dx-viewport` class to the `body` tag:
+
+    <!--HTML-->
+        <body class="dx-viewport">
+
+## 5. Add a new component
+
+1. Create a new file for the component --- `src/components/CatFactGrid.vue`. Populate it with the following two tags:
+
+    <!--TypeScript-->
+        <script setup lang="ts">
+            // component setup
+        </script>
+        <template>
+            // component declaration
+        </template>
+
+2. Modify the `src/App.vue` file to display our new component:
+
+    <!--TypeScript-->
+        <script setup lang="ts">
+        import CatFactGrid from './components/CatFactGrid.vue'
+        </script>
+
+        <template>
+        <CatFactGrid />
+        </template>
+
+## 6. Configure the Data Grid
+
+Go back to the `CatFactGrid.vue` file.  The component is currently empty. Use the `script` tag to set up a DevExtreme Data Grid:
+
+<!--TypeScript-->
+    <script setup lang="ts">
+        import { DxDataGrid } from "devextreme-vue/data-grid"; // Import the DevExtreme Data Grid component
+        import CustomStore from "devextreme/data/custom_store"; // Import the CustomStore object
+
+        const catFactStore = new CustomStore({ // Define a CustomStore that pulls data from the Cat Facts API
+        load: () => {
+            return fetch("https://catfact.ninja/facts")
+            .then(handleErrors)
+            .then((response) => response.json())
+            .catch(() => {
+                throw "Network error";
+            });
+        },
+        });
+
+        function handleErrors(response: Response) { // Handle failing network requests
```

Comment:
`// Handle failing network requests`

Are requests still in the process of "failing", or do you mean "Handle failed network requests"?